Dockerfile

```diff
@@ -1,5 +1,6 @@
 FROM openjdk:8-jdk-alpine
 WORKDIR /
-COPY target/conter-0.0.1-SNAPSHOT.jar app.jar
+COPY conter-0.0.1-SNAPSHOT.jar app.jar
+COPY script.sh script.sh
```

Comment:
Are you creating the script in your Jenkins job? Ideally it should be committed to this repository as part of this PR.